napari/layers/labels/labels.py

```diff
@@ -226,6 +227,8 @@ class Labels(_ImageBase):
         background label `0` is selected.
     """
 
+    events: EmitterGroup
```

Comment:
Why here and not directly in `Layer`?

Reply:
There is some problem with inheritance types of parent parent class.